tools/check

```diff
@@ -241,6 +241,21 @@ should_run_build_runner() {
     return 1
 }
 
+run_l10n() {
+    local output_path=lib/generated/l10n
+
+    files_check "${output_path}"/'*'.dart \
```

Comment:
I think this check is too narrow and will miss some cases where this suite should run.

For examples, look back at the other `files_check` call sites.

This should also include an "Omitted from this check" comment like the ones at those other call sites, listing all the other files in the repo that would belong in a 100% version of this check (because changes to them could affect whether this suite passes) but which we're choosing to leave out as described at the `files_check` doc comment. I put those there partly as a way of making sure I explicitly thought through what files could matter to each check, to try to avoid this class of bug :slightly_smiling_face: